CMakeLists.txt

```diff
@@ -343,26 +343,33 @@ if(SERIALBOX_TESTING_GRIDTOOLS)
   
 endif(SERIALBOX_TESTING_GRIDTOOLS)
 
-#---------------------------------------- Boost ----------------------------------------------------
-# Workaround for Cray
-get_property(SHARED_LIB_SUPPORT GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS)
-if (SHARED_LIB_SUPPORT)
-    set(Boost_USE_STATIC_LIBS OFF)
-    set(Boost_USE_STATIC_RUNTIME OFF)
-    set(Boost_USE_MULTITHREADED ON)
-    add_definitions(-DBOOST_ALL_DYN_LINK)
-    add_definitions(-DBOOST_LOG_DYN_LINK)
+#---------------------------------------- std::experimental::filesystem ----------------------------
+find_library( STD_FILESYSTEM_LIB NAMES stdc++fs PATHS

```

Comment:
This really hacky but I guess it works for the time.. maybe create an issue to update it once `std::filesystem` becomes more common (I guess at some point it will be merged in `libc++` or `libstdc++` making the detection obsolete)

Reply:
Yes, it is, but there seems to be no proper way in CMake to detect the library. The one which works on CSCS machines is the ENV thing. So we could limit it to this and people would have to adapt there `LD_LIBRARY_PATH` if they want to use this feature...